src/Components/test/Microsoft.AspNetCore.Components.Test/RendererTest.cs

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.AspNetCore.Components.RenderTree;
+using Microsoft.AspNetCore.Components.Test.Helpers;
```

Comment:
Should we create an editor config so these things stop changing? Our eng guidelines says system first.

Reply:
Copied over the .editorconfig from MVC. I'll follow up with engg to commonize all the .editorconfigs